### DISCUSSION TITLE
how to define glyph sets?

### DISCUSSION THREAD

**Discussion by bdenckla** [fontforge/fontforge#4723] [Q&A]
I'm probably missing something, but I only see two mechanisms in FontForge to define glyph sets, both of which are somewhat special-purpose:

1. MarkClasses (mark attachment classes)
2. MarkSets (mark filtering sets)

Is there way to define general-purpose glyph sets in FontForge, for use in defining lookups?

Consider the following feature file language example:
```
@glyphclass1=[glyphname1 glyphname2];
```
Is there a mechanism in FontForge corresponding to the syntax used in this example?

**Comment by skef** [comment]
I could be wrong but I don't believe there is an entirely generic mechanism for this, no. 

It's possible to use FontForge with feature files rather than its internal lookup representation, (although it is more convenient with some extra Tools menu options).
